README.md

```diff
@@ -254,23 +220,21 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
Great job @principlebrothers 🤝

- [ ] As stated in the project requirements, kindly give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).
- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.

Reply:
Thanks @1GORDON for the feedback. I will implement the changes right away